charts/cubestore/templates/router/service.yaml

```diff
@@ -18,9 +18,6 @@ metadata:
     {{- end }}
   {{- end }}
 spec:
-  {{- range $key, $val := .Values.router.service.spec }}
-  {{ $key }}: {{ $val | quote }}
-  {{- end }}
```

Comment:
@lvauvillier , 
May I ask why these 3 lines were removed? 
I want to understand the reasoning behind the change as it is necessary for creating an Internal LoadBalancer in GCP ([GCP guide describing the internal load balancer manifest](https://cloud.google.com/kubernetes-engine/docs/how-to/internal-load-balancing#create)). Happy to make changes that are consistent with this chart but, I would like to understand why they were removed.